db/migrate/20161207095801_remove_permissions_default_value.rb

```diff
@@ -0,0 +1,5 @@
+class RemovePermissionsDefaultValue < ActiveRecord::Migration[5.0]
+  def change
+    change_column :users, :permissions, :string, default: nil
```

Comment:
It's to do with the YAML deserialising. 
Without the field defaults to an empty string which does NOT deserialise to an Array as expected.